changelog/0.28.0_2020-12-04/text-editor-file-path-label

```diff
@@ -1,5 +1,5 @@
 Bugfix: Don't break file/folder names in text editor
 
-The label in the text editor that displays the path of the active file was removing the first character instead of trimming leading slashes. This might have lead to situations where actual characters were removed. We fixed this by only removing leading slahes instead of blindly removing the first character.
+The label in the text editor that displays the path of the active file was removing the first character instead of trimming leading slashes. This might have lead to situations where actual characters were removed. We fixed this by only removing leading slashes instead of blindly removing the first character.
```

Comment:
For most of the repositories I looked at, I actively skipped the changelog/ child directories, apparently I didn't for this repository. I can trivially drop the directory if requested.

Reply:
Doesn't do any harm, we're happy about the fixes/contribution!